spec/models/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validations' do
+    context 'when passing too long password' do
+      it 'validates password length in bytes' do
+        password = "#{'p' * 72}assword"
```

Comment:
Nice but a bit opaque, and what's even more important the condition in your validator states `<= 72`.
The length of a password is rarely a business rule, so here it'd be acceptable but consider this:
Requirements live in the test suite (that is if there's some business logic - e.g. an email is to be sent exactly 48h before a movie starts - it should be covered in a test).
Given that the tests should guard us against code changes that might cause stepping over the line of requirements, the code of your validator is a bit imprecise - it introduces some play, that in case of changes to the validator will go unnoticed (someone might change it to `<= 77` and it'll pass, because the test tells that your requirement is `bytesize less than 79`).

So if the requirement is `<= 72` then I'd suggest changing it to `"x" * 73`.

Reply:
Thank you for your remark, I will change it as suggested:)